src/model.py

```diff
@@ -2153,7 +2101,7 @@ def forward(self, batch):
             attention_mask = attention_mask,
         ).last_hidden_state
         embeddings = self.bert_dropout(embeddings).permute(0, 2, 1)
-        sentence_output = embeddings.clone()
+        shared_output = embeddings.clone()
```

Comment:
Changed name to better match naming in task-specific loops (for better human readability)